web-components/src/components/inputs/new/ImageField/ImageField.tsx

```diff
@@ -16,14 +16,15 @@ import { useImageFieldStyles } from './ImageField.styles';
 interface Props {
   name: string;
   label?: string;
+  description?: string;
   buttonText?: string;
   initialFileName?: string;
   disabled?: boolean;
   optional?: boolean | string;
   circularCrop?: boolean;
   fixedCrop?: Partial<Crop>;
   children: ReactNode;
-  setValue: (value: string) => void;
+  setValue: (value: any) => void;
```

Comment:
why do we need to switch to `any`?

Reply:
This function used to receive different types of values, I change it back so the value is always a string but forget to change the type too, will update.